Updated sponsor page and added pentland thank you

leticiacid0478 opened the pull request:
- made slideshow not overlap with mobile menu by adding a z index
- added pentland's to slideshow

Judge24601 commented on index.html:
```diff
@@ -124,7 +124,7 @@ <h3 class="mbr-section-title mbr-fonts-style mb-3 display-5">
                     <img src="assets/images/front4.jpeg" alt="Testing GRASP functionality">
                     <p class="mbr-description mbr-fonts-style pt-2 align-center display-4"><em>
-                    Our pilot testing GRASP's functionality.</em></p>
+                    Our pilot testing GRASP's functionality. CHange here</em></p>
```
why did you do this

Judge24601 replied:
because i wanted to

Judge24601 commented on index.html:
```diff
@@ -146,7 +146,7 @@ <h3 class="mbr-section-title mbr-fonts-style mb-3 display-5">
                 <h3 class="mbr-section-title mbr-fonts-style mb-4 display-2">
                     <strong>GRASP</strong></h3>
-                <h4 class="mbr-section-subtitle mbr-fonts-style mb-4 display-5">Our current project: GRASP</h4>
+                <h4 class="mbr-section-subtitle mbr-fonts-style mb-4 display-5">Change Here. Our current project: GRASP</h4>
```
this is a terrible idea

leticiacid0478 committed:
Updated sponsor page and added pentland thank you

- made slideshow not overlap with mobile menu by adding a z index
- added pentland's to slideshow
